mapbox/validation.py

```diff
@@ -0,0 +1,31 @@
+
+class MapboxValidationError(ValueError):
+    pass
+
+
+class InvalidPlaceTypeError(MapboxValidationError):
```

Comment:
The dict is just an implementation detail of how we validate. 

A ValueError is "Raised when a built-in operation or function receives an argument that has the right type but an inappropriate value" - user input a string but that string was inappropriate. They don't need to know there's a dictionary underneath, do they?